pages/hub/apps/sdk/write-hooks/useDeBridgeBet.mdx

```diff
@@ -0,0 +1,165 @@
+import { Callout } from 'components';
+
+This hook is used for users to place bets from others blockchains. For instance, if a user desires to wager on Polygon but lacks tokens for it, they can utilize tokens from Arbitrum instead.
```

Comment:
This hook enables users to place bets using tokens from different blockchains. For example, if someone wants to bet on Polygon but does not have the necessary tokens, they can opt to use tokens from the Arbitrum (or any other supported network) instead.

Reply:
7983ca0f7e6ba2f1727dfab481f4ca41c4bede58